src/main/main/java/org/example/PetFeeding.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class PetFeeding {
+    @Configuration
```

Comment:
It's not necessary here. This annotation should present on the class only, like above